qwandaq/src/main/java/life/genny/qwandaq/utils/CacheUtils.java

```diff
@@ -29,12 +41,13 @@
 @RegisterForReflection
 public class CacheUtils {
 
-	static final Logger log = Logger.getLogger(CacheUtils.class);
-
 	static Jsonb jsonb = JsonbBuilder.create();
 
 	private static GennyCache cache = null;
 
+	@Inject
+	private static final Logger log = Logger.getLogger(CacheUtils.class);
```

Comment:
does @Inject on a static final object work? This definitely works as `@Inject Logger log;`